packages/react-components/src/Concepts/MigratingFromV8.stories.mdx

```diff
@@ -0,0 +1,158 @@
+| `FluentProvider` | [`@fluentui/react-provider`](https://github.com/microsoft/fluentui/tree/master/packages/react-provider)   |
+| `Link`           | [`@fluentui/react-link`](https://github.com/microsoft/fluentui/tree/master/packages/react-link)           |
+| `Menu`           | [`@fluentui/react-menu`](https://github.com/microsoft/fluentui/tree/master/packages/react-menu)           |
+| `MenuButton`     | [`@fluentui/react-button`](https://github.com/microsoft/fluentui/tree/master/packages/react-button)       |
+| `Popover`        | [`@fluentui/react-popover`](https://github.com/microsoft/fluentui/tree/master/packages/react-popover)     |
+| `Portal`         | [`@fluentui/react-portal`](https://github.com/microsoft/fluentui/tree/master/packages/react-portal)       |
+| `Text`           | [`@fluentui/react-text`](https://github.com/microsoft/fluentui/tree/master/packages/react-text)           |
+| `ToggleButton`   | [`@fluentui/react-button`](https://github.com/microsoft/fluentui/tree/master/packages/react-button)       |
+| `Tooltip`        | [`@fluentui/react-tooltip`](https://github.com/microsoft/fluentui/tree/master/packages/react-tooltip)     |
+
+---
+
+#### "Anything else I need to know about the file structure moving to the new version?"
+
+Well, there are actually a couple of things that have changed in version 9.
+
+First of all, we are taking greater strides towards separating the concerns of a component into different files via the use of hooks. In version 8 there was already some semblance of that by separating things into the following structure for any given component:
+
+- All type information in a `[ComponentName].types.ts` file.
+- All styling information in a `[ComponentName].styles.ts` file.
+- The rest of the component logic in a `[ComponentName].base.tsx` file.
+- Bringing all of them up together in a `[ComponentName].tsx` file.
+
+In version 9 the structure for a given component would be something more like:
+
+- All type information in a `[ComponentName].types.ts` file.
+- All rendering logic in a `render[ComponentName].tsx` file.
+- All styling information in a `use[ComponentName]Styles.ts` file.
+- All state management in a `use[ComponentName]State.ts` file.
+- Providing defaults and resolving shorthands in a `use[ComponentName].ts` file (keep reading to learn more about what shorthands are).
+- Bringing all of them up together in a `[ComponentName].tsx` file.
+
+In this way, we can actually maintain the separation of concerns we had for types and styling before while also separating the concerns of state management and rendering logic into their own individually reusable parts.
+
+Another added benefit in version 9 is that, because of the move to scaffolded packages, we are able to collocate all our component examples in their respective package next to the component's code instead of having them in the separate `@fluentui/react-examples` package.
```

Comment:
I've removed the mention of `benefit` and `because of the move to scaffolded packages` and just referred to the fact that we're collocating examples as a change in v9 to avoid adding the extra background that is probably not needed without having any inaccurate information in there.